wodles/aws/buckets_s3/waf.py

```diff
@@ -20,6 +29,22 @@ class AWSWAFBucket(AWSCustomBucket):
     def __init__(self, **kwargs):
         kwargs['db_table_name'] = 'waf'
         AWSCustomBucket.__init__(self, **kwargs)
+        if self.check_waf_type():
+            self.service = 'WAFLogs'
+            self.type = "WAFNative"
+        else:
+            self.type = "WAFKinesis"
```

Comment:
Create two constants for these types so we avoid having strings replicated across the code. Also, let's try to use single or double quotes for strings, but not both.

```suggestion
            self.type = 'WAFNative'
        else:
            self.type = 'WAFKinesis'
```